src/examples/eet/eet-data-cipher_decipher.c

```diff
@@ -19,55 +19,57 @@ main(void)
    const char *key = "This is a crypto key";
    const char *key_bad = "This is another crypto key";
 
-   char *file = strdup("/tmp/eet_cipher_example_XXXXXX");
    Eet_File *ef;
    char *test;
    int size;
+   Eina_Tmpstr *tmpfile = NULL;
    int tmpfd;
 
    eet_init();
 
-   if (-1 == (tmpfd = mkstemp(file)) || !!close(tmpfd))
+   /* tmpfile will be created in temporary directory (with eina_environment_tmp) */
+   tmpfd = eina_file_mkstemp("eet_cipher_example_XXXXXX", &tmpfile);
+   if ((-1 == tmpfd) || !!close(tmpfd))
      {
         fprintf(
-                stderr, "ERROR: could not create temporary file (%s) : %s\n",
-                file, strerror(errno));
+                stderr, "ERROR: could not create temporary tmpfile (%s) : %s\n",
+                tmpfile, strerror(errno));
         goto panic;
      }
 
    /* Crypt an eet file. */
-   ef = eet_open(file, EET_FILE_MODE_WRITE);
+   ef = eet_open(tmpfile, EET_FILE_MODE_WRITE);
    if (!ef)
      {
         fprintf(
-          stderr, "ERROR: could not access file (%s).\n", file);
+          stderr, "ERROR: could not access tmpfile (%s).\n", tmpfile);
         goto error;
      }
 
    if (!eet_write_cipher(ef, "keys/tests", buffer, strlen(buffer) + 1, 0, key))
      {
         fprintf(
-          stderr, "ERROR: could not access file (%s).\n", file);
+          stderr, "ERROR: could not access tmpfile (%s).\n", tmpfile);
         goto error;
      }
 
    eet_close(ef);
 
    /* Decrypt an eet file. */
-   ef = eet_open(file, EET_FILE_MODE_READ);
+   ef = eet_open(tmpfile, EET_FILE_MODE_READ);
    if (!ef)
      {
         fprintf(
-          stderr, "ERROR: could not access file (%s).\n", file);
+          stderr, "ERROR: could not access tmpfile (%s).\n", tmpfile);
         goto error;
      }
 
    test = eet_read_cipher(ef, "keys/tests", &size, key);
    if (!test)
      {
         fprintf(
-          stderr, "ERROR: could decript contents on file %s, with key %s.\n",
-          file, key);
+          stderr, "ERROR: could decript contents on tmpfile %s, with key %s.\n",
```

Comment:
I think this part wasn't supposed to be changed to "tmpfile".

Reply:
```suggestion
          stderr, "ERROR: could decript contents on file %s, with key %s.\n",
```